docs/internals/contributing/writing-code/javascript.txt

```diff
@@ -0,0 +1,80 @@
+==========
+JavaScript
+==========
+
+While most of Django core is Python, the ``admin`` and ``gis`` contrib apps contain JavaScript code.
+
+Please follow these coding standards when writing JavaScript code for inclusion in Django.
+
+Code style
+----------
+
+- Please conform to the indentation style dictated in the ``.editorconfig`` file.  We recommend using a text editor with `EditorConfig`_ support to avoid indentation and whitespace issues.  Most of the JavaScript files use 4 spaces for indentation, but there are some exceptions
```

Comment:
That's fine. Could you please wrap lines at 79 characters and use 1 space between sentences as well. I think we can remove the sentence, "Most of the JavaScript files use 4 spaces for indentation, but there are some exceptions" right?